### DISCUSSION TITLE
What will be the best way to configure the LLDP log creation ?

### DISCUSSION THREAD

**Discussion by juacostab** [lldpd/lldpd#467] [Q&A]
Hello,

I am building a discovery agent using LLDP. I need for RCA in to include in my diagnostics module  snippets of the LLDP log.

How do you recommend, configuring the log? I was looking at the logger code and it seems it supports syslog, but did not find a way to set the configuration on the configuration file or the demon init file.

Any  help will be gladly appreciated

**Comment by vincentbernat** [comment]
It supports only the local syslog daemon (through `/dev/log`), so there is no configuration knob. You need to use a local syslog daemon to forward or store the logs.

**Reply by juacostab** [reply]
@vincentbernat thanks a lot for the response. I definitely can do that
